wdl/tasks/GATKBestPractice.wdl

```diff
@@ -266,11 +266,12 @@ task MergeBamouts {
 
     Int disk_size = ceil(size(bams, "GiB") * 2) + 10
 
-    command {
+    command <<<
```

Comment:
Let's leave this alone.
I cannot recall what the state of this pipeline is in, but we'll most likely take this pipeline out completely.

Reply:
👍